data/ThorOnTheRocks.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Gianluca Galota",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/ThorOnTheRocks.png",
+  "tags": ["javascript", "typescript", "react", "node", "open source", "design system"],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/GianlucaGalota" },
+    { "platform": "github", "url": "https://github.com/ThorOnTheRocks" }
+  ],
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ThorOnTheRocks",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/GianlucaGalota",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/gianluca-galota/",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Joined EddieHub",
+      "date": "November 2022",
+      "icon": "github",
+      "color": "blue",
+      "description": "Made my first PR to EddieHub and my first contribution to open source",
+      "url": "https://github.com/EddieHubCommunity/LinkFree"
+    },
+    {
+      "title": "Started my journey as Frontend Developer",
+      "date": "November 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "First job as Frontend Developer",
+      "url": "https://thorontherocks.github.io/personal-website/"
+    },
+    {
+      "title": "Started learning to code!",
+      "date": "April 2021",
+      "icon": "github",
+      "color": "blue",
+      "description": "Started learning to code with the Le Wagon curriculum! Developed NomadConnect my first full-stack project.",
+      "url": "https://www.nomadconnect.xyz/"
+    }
+  ]
+}
```

Comment:
I think the GitHub Action check is failing because the file is not ending with a blank like (and no spaces), as highlight here by GitHub

Reply:
Thanks you so much for checking my commit @eddiejaoude. I made a new one but it seems that is still failing the check...not sure why